File: mt/mt1051/week09/HW23/HW23_109601003.go
/*
Created Date: 2023/11/20
HW: 23
Author: 林群賀
Student Number: 109601003
*/

package main

import "fmt"

func countDepreciation() {
	initValue := 100000
	rate := 9000

	for i := 1; i <= 10; i++ {
		depreciationExpense := rate
		accumulatedDepreciation := rate * i
		endOfYearValue := initValue - rate*i

		fmt.Printf("Year %d, Depreciation expense: %d, Accumulated depreciation: %d, End of year value: %d\n", i, depreciationExpense, accumulatedDepreciation, endOfYearValue)
	}
}

func main() {
	countDepreciation()
}

File: mt/mt1051/week09/HW23/HW23_109601003.py
# -*- coding: utf-8 -*-
"""
Date: 2023/11/20
HW: 23
Author: 林群賀
Student Number: 109601003
"""

def count_depreciation():
    init = 100000
    rate = 9000

    for i in range(1, 11):
        print(f"年度 {i}, 折舊費用為 {rate}, 累積折舊為 {rate * i}, 期末帳面價值為 {init - rate * i}")

def main():
    count_depreciation()

if __name__ == "__main__":
    main()
